README.md

```diff
@@ -1,116 +1,184 @@
-# zipgateway
+# Z/IP Gateway SDK
```

Comment:
Remove the SDK mention
```suggestion
# Z/IP Gateway
```

Reply:
I aligned to RN title , but I prefer without too